### PR TITLE
Extraction-db: Extract data from db

### PR DESCRIPTION
**Pull request by araiara**

### CODE REVIEW

File: schema/create_raw_employee_data.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    manager_employee_id VARCHAR(500),
+    employee_role VARCHAR(500),
+    salary VARCHAR(500),
+    hire_date VARCHAR(500),
+    terminated_date VARCHAR(500),
+    terminated_reason VARCHAR(500),
+    dob VARCHAR(500),
+    fte VARCHAR(500),
+    location VARCHAR(500)
+);
```
Comment (nikitashrestha) [inline schema/create_raw_employee_data.sql]: Why do we have the same table schema?

File: schema/create_raw_sales_data.sql [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    product_id VARCHAR(500),
+    product_name VARCHAR(500),
+    category_id VARCHAR(500),
+    category_name VARCHAR(500),
+    current_price VARCHAR(500),
+    sold_price VARCHAR(500),
+    sold_quantity VARCHAR(500),
+    remaining_quantity VARCHAR(500),
+    sales_date VARCHAR(500)
+);
```
Comment (nikitashrestha) [inline schema/create_raw_sales_data.sql]:
```suggestion
CREATE TABLE IF NOT EXISTS raw_sales_data (
  user_id VARCHAR(500),
  username VARCHAR(500),
  product_id VARCHAR(500),
  product_name VARCHAR(500),
  category_id VARCHAR(500),
  category_name VARCHAR(500),
  current_price VARCHAR(500),
  sold_price VARCHAR(500),
  sold_quantity VARCHAR(500),
  remaining_quantity VARCHAR(500),
  sales_date VARCHAR(500)
);

```

Comment (nikitashrestha) [reply schema/create_raw_sales_data.sql]: Apply same changes to all SQL quries. And please follow double space indentation

File: schema/create_raw_timesheet_data_archive.sql [hunk 3]
```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS raw_timesheet_data_archive (
+	employee_id	VARCHAR(500),
+	cost_center	VARCHAR(500),
+	punch_in_time VARCHAR(500),	
+	punch_out_time VARCHAR(500),
+	punch_apply_date VARCHAR(500),
+	hours_worked VARCHAR(500),
+	paycode VARCHAR(500),
+    file_name VARCHAR(500)
```
Comment (nikitashrestha) [inline schema/create_raw_timesheet_data_archive.sql]: Won't it be better if we add file date too?

File: src/pipeline/extract_employee_data_from_db.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
+from src.utils import *
+
+def extract_employee_data():
+    create_table_schema('destination_employee', 'employee_timesheet_db') # create employee destination table
```
Comment (nikitashrestha) [inline src/pipeline/extract_employee_data_from_db.py:4]: Do we really need to create schema each time?

File: schema/create_raw_employee_data.sql (hunk 1 above)
Comment (araiara) [reply schema/create_raw_employee_data.sql]: Didi, I created a branch from hijo ko extraction, so these are the files from yesterday's task.

**Pull request merged**